pkg/kubelet/kubelet_server_journal.go

```diff
@@ -112,36 +112,6 @@ func newJournalArgsFromURL(query url.Values) (*journalArgs, error) {
 	}, nil
 }
 
-// Args returns the journalctl arguments for the given args.
-func (a *journalArgs) Args() []string {
-	args := []string{
-		"--utc",
-		"--no-pager",
-	}
-	if len(a.Since) > 0 {
-		args = append(args, "--since="+a.Since)
-	}
-	if len(a.Until) > 0 {
-		args = append(args, "--until="+a.Until)
-	}
-	if a.Tail > 0 {
-		args = append(args, "--pager-end", fmt.Sprintf("--lines=%d", a.Tail))
-	}
-	if len(a.Format) > 0 {
-		args = append(args, "--output="+a.Format)
-	}
-	for _, unit := range a.Units {
-		if len(unit) > 0 {
-			args = append(args, "--unit="+unit)
-		}
-	}
-	if len(a.Pattern) > 0 {
-		args = append(args, "--grep="+a.Pattern)
-		args = append(args, fmt.Sprintf("--case-sensitive=%t", a.CaseSensitive))
-	}
-	return args
-}
-
```

Comment:
please minimize the patch. The removal is cosmetics and has a chance to conflict on rebases.

Reply:
I don't see this as merely cosmetical, as this func is linux-only, assembling the args for the journalctl cmd. That's why I moved it into a file that only gets build on Linux, with other implementations of the same function for other OSes in other files (i.e the right implementation is chosen at compile time).

Another option that also works is to switch on  `runtime.GOOS`, which makes the diff smaller. 

Please elaborate

Reply:
I compiles on every platform, doesn't it? That's my definition of cosmetical here.

Reply:
ic, getLoggingCmd on Windows is completely different.